databus-util-cmdline/databus-util-cmdline-impl/src/main/java/com/linkedin/databus/util/FieldToAvro.java

```diff
@@ -202,7 +202,7 @@ else if (typeInfo instanceof CollectionTypeInfo)
     field.put("default", null);
 
     // Field type
-    String[] type = new String[] { "null", typeInfo.getPrimitiveType().getAvroType() };
+    String[] type = new String[] { typeInfo.getPrimitiveType().getAvroType(), "null"};
```

Comment:
with using contains function , I think this change can make it run faster. I test the 'column is null case',and it runs right. I didn't consider the default value case ,I commit and change it back.Thank you.